src/rdf4cpp/rdf/datatypes/registry/util/CharConvExt.hpp

```diff
@@ -19,7 +19,7 @@ namespace rdf4cpp::rdf::datatypes::registry::util {
  * @return the resulting value
  * @throws std::runtime_error if the string cannot be parsed
  */
-template<std::integral I>
+template<std::integral I, ConstexprString datatype>
```

Comment:
Please document the template parameters here and in DateTimeUtils. Can be adressed together with making the `ConstexprString datatype` parameters more robust.

Reply:
moved to error message issue